tests/framework/installer/ceph_installer.go

```diff
@@ -352,6 +352,7 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(helmInstalled bool, systemNa
 	logger.Infof("Uninstalling Rook")
 	var err error
 	for _, namespace := range namespaces {
+		h.k8shelper.PrintPodDescribeForNamespace(namespace)
```

Comment:
this will describe all pods in the namespace? Do we need to print this during every suite teardown or can we restrict it to some failure condition? Seems too verbose for every run.

Reply:
This is a good thought. The intention was just for teardown after failure conditions, but this was the quickest way for me just to see output. I'll look around some more to see if there is a better place to put this.

A good place for might also be right before the test panics on timeout, as it appeared to me that the timeout panic didn't result in teardown of the cluster.